src/components/navbar.js

```diff
@@ -0,0 +1,110 @@
+import React from "react"
+import { Link } from "gatsby"
+import {
+  Collapse,
+  Navbar,
+  NavbarToggler,
+  Nav,
+  NavItem,
+  NavLink,
+} from "reactstrap"
+
+import Logo from "../images/svg/logo_sinf_simp.inline.svg"
+import NavbarStyles from "../styles/navbar.module.css"
+
+// Markup for navbar links redirecting to an internal page
```

Comment:
Remove comment and rename function to something like `NavbarLink`? Seems clear enough to me and disregards a comment (which is always nice)